src/main/scala/com/github/danielwegener/intellij/cucumber/scala/CucumberScalaExtension.scala

```diff
@@ -52,8 +53,16 @@ class CucumberScalaExtension extends AbstractCucumberExtension {
 
   @NotNull override def getGlues(@NotNull file: GherkinFile, jGluesFromOtherFiles: JavaSet[String]): JavaCollection[String] = {
     // never called? wtf
-    LOG.debug("GET GLUES CALLED")
-    Collections.emptyList()
+    import scala.collection.JavaConverters._
+
+    val gluePaths = glueClassesFromFeature(file).map(_.getPath).distinct
+    val otherGlues = jGluesFromOtherFiles.asScala
+
+    LOG.warn(s"""GET GLUES CALLED found glues: ${gluePaths.mkString(":")} known glues: ${otherGlues.mkString(":")}""" )
```

Comment:
Does this need to be a warn? If the `mkString` can not be made lazy, it should at least be behind a `if (LOG.isActive(LEVEL))`

Reply:
Doh! sorry not it can be a debug...